third_party/terraform/tests/data_source_google_compute_ssl_certificate_test.go

```diff
@@ -0,0 +1,82 @@
+package google
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+func TestAccDataSourceComputeSslCertificate(t *testing.T) {
+	t.Parallel()
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:  func() { testAccPreCheck(t) },
+		Providers: testAccProviders,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccDataSourceComputeSslCertificateConfig(),
+				Check: resource.ComposeTestCheckFunc(
+					testAccDataSourceComputeSslCertificateCheck("data.google_compute_ssl_certificate.cert", "google_compute_ssl_certificate.foobar"),
+				),
+			},
+		},
+	})
+}
+
+func testAccDataSourceComputeSslCertificateCheck(dataSourceName string, resourceName string) resource.TestCheckFunc {
```

Comment:
I recently submitted a PR that adds a helper fn for these sorts of checks so they don't have to be totally written out. Mind updating to use it? https://github.com/GoogleCloudPlatform/magic-modules/pull/1780

Reply:
Done - that's shifted away a fair wad of boilerplate. Thanks.